src/style.css

```diff
@@ -0,0 +1,25 @@
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
last child shouldn't have margin-right. Use `:not(:last-child)`

Reply:
Hi! Thank you very much :) 

Isn't it easier, however, to use a `:last-child?` Like this:

`.stars__star:last-child {
  margin-right: 0;
}`